### PR TITLE
Adds documentation for extending json-ld contexts

### PR DESCRIPTION
**Pull request by maechler**

### CODE REVIEW

File: core/extending-jsonld-context.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Extending JSON-LD context
+
+API Platform Core provides the possibility to extend the JSON-LD context of properties. This allows you to describe JSON-LD 
+typed values, inverse properties using the @reverse keyword and you can even overwrite the @id property this way. Everything you define
```
Comment (dunglas) [inline core/extending-jsonld-context.md]: `@reverse`

(same of `@id`)

File: core/extending-jsonld-context.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      "someProperty": {
+        "a": "textA",
+        "b": "textB"
+      }
+    }
+  }
+}
+```
+
+Note that you do not have to provide the @id attribute, if you do not provide an @id attribute, the value from "iri" will be taken.
```
Comment (dunglas) [inline core/extending-jsonld-context.md]: Same here.

**Comment by dunglas** [comment]
Great! I just left some minor comments.

**Comment by maechler** [comment]
@dunglas Thanks, I updated the pull request.

**Comment by dunglas** [comment]
Thank you @maechler

**Pull request merged**